include/IFileSystem.h

```diff
@@ -239,6 +241,37 @@ class IFileSystem : public virtual core::IReferenceCounted
 	\return True if file exists, and false if it does not exist or an error occured. */
 	virtual bool existFile(const path& filename) const =0;
 
+
+	template<typename StringUniqueType>
+	inline core::smart_refctd_ptr<asset::ICPUBuffer> loadBuiltinData()

```

Comment:
`core::ICPUBuffer` you dont want a circular dependency of `asset` namespace on the `io` namespace

Reply:
since `core::ICPUBuffer` does not exist, I will replace this with `core::IBuffer`

Reply:
good